### ISSUE TITLE
SerializationException: The value "parent" cannot be parsed as a type "Boolean"

### ISSUE THREAD

**Issue by dartrax** [XiaoFaye/WooCommerce.NET#722]
- Wordpress version 6.1.1, WooCommerce version 7.2.2 and WooCommerce.NET version 0.8.4

This works:
```
Dim p = Await Woo.Product.GetAll(New Dictionary(Of String, String) From {{"slug", "my-product-slug"}})
Process.Start(p.First.permalink)
```

This does not work:
```
Dim p = Await Woo.Product.GetAll(New Dictionary(Of String, String) From {{"sku", "12345678"}})
Process.Start(p.First.permalink)
```

In my test both calls to GetAll should deliver the same product (my-product-slug is the slug of the product with the sku 12345678)

The error is:
```
System.Runtime.Serialization.SerializationException: "Error deserializing object "of type System.Collections.Generic.List`1[[WooCommerceNET.WooCommerce.v3.Product, WooCommerce.NET, Version=0.8.4.0, Culture=neutral, PublicKeyToken=ff41d0b6c937438d]]". The value "parent" cannot be parsed as type "Boolean"."

XmlException: the value "parent" cannot be parsed as type "Boolean".

FormatException: The string 'parent' not a valid Boolean value.
```

**Comment by dartrax** [comment]
This bug is still present in version 0.8.5

**Comment by XiaoFaye** [comment]
I don't get this error on local testing, do you mind uploading the returned json details so I can check?

**Comment by dartrax** [comment]
Thanks for looking into it. I've uploaded only an excerpt of the json, please let me know if you need the whole thing. I believe this is the culprit:
```
... 
"tax_status":"taxable","tax_class":"","manage_stock":"parent","stock_quantity":12,"backorders":"no",
...
```
So manage_stock isn't boolean, it has the value "parent", probably because the SKU I'm requesting is a product variant which shares stock with the parent:

![image](https://user-images.githubusercontent.com/15105943/211617851-c79b5c64-53da-4d38-9b55-2903a07e6987.png)

**Comment by Mattoia90** [comment]
I have the same problem with Variation products.
When I use Postman I see < "manage_stock":"parent" > but in the class product is typed:  bool?

**Comment by XiaoFaye** [comment]
It's defined as bool in WC's document, however when in variation products, it could be a string with value "parent". Will fix this in the next version.

**Comment by XiaoFaye** [comment]
@dartrax This is fixed in version 0.8.6.
